internal/pkg/build/sources/conveyorPacker_oci.go

```diff
@@ -440,6 +444,20 @@ func (cp *OCIConveyorPacker) insertEnv() (err error) {
 	return nil
 }
 
+func (cp *OCIConveyorPacker) insertOCILabels() (err error) {
```

Comment:
Note - we are passing labels through to the next stage of the build via the labels.json file in the rootfs so things fit in with the existing `metadata.go` handling of labels from base images.

It's not particularly pretty, but it's consistent with the flow for other sources. Keeping metadata out of band w.r.t. the rootfs is probably something to tackle wholesale in a major version bump.